pennylane/measure.py

```diff
@@ -123,3 +177,48 @@ def sample(op):
         qml._current_context._append_op(op)
 
     return op
+
+
+def probs(wires):
+    r"""Probability of each computational basis state.
+
+    This measurement function accepts no observables, and instead
+    instructs the QNode to return a flat array containing the
+    probabilities of each quantum state.
+
+    Marginal probabilities may also be requested by restricting
+    the wires to a subset of the full system.
+
+    **Example:**
+
+    .. code-block:: python3
+
+        dev = qml.device("default.qubit", wires=2)
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.Hadamard(wires=1)
+            qml.CNOT(wires=[0, 1])
+            return qml.probs(wires=[0, 1])
+
+    Executing this QNode:
+
+    >>> circuit()
+    array([0.5, 0.5, 0. , 0. ])
+
+    The returned array is in lexicographic order, so corresponds
+    to a :math:`50\%` chance of measuring either :math:`|00\rangle`
+    or :math:`|01\rangle`.
+
+    Args:
+        wires (Sequence[int] or int): the wire the operation acts on
+    """
+    # pylint: disable=protected-access
+    op = qml.Identity(wires=wires, do_queue=False)
```

Comment:
Currently ``qml.Identity`` inherits from ``CVObservable``, how come this works in the qubit case? Would have expected that there is a check somewhere that would throw an error

Reply:
It looks the `qnodes/base.py` explicitly makes an exception for `Identity`:
```python
        # True if op is a CV, False if it is a discrete variable (Identity could be either)
        are_cvs = [
            isinstance(op, CV) for op in self.queue + list(res) if not isinstance(op, qml.Identity)
        ]
```